cutil/script/script.go

```diff
@@ -53,7 +53,7 @@ func buildBootstrapSetupScript(cluster *cluster.Cluster, dir, file string) ([]by
 	if err != nil {
 		return nil, err
 	}
-	script := []byte("mkdir -p " + dir + "\nsudo sh -c 'cat <<EOF > " + dir + "/" + file + "\n")
+	script := []byte("mkdir -p " + dir + "\ncat <<\"EOF\" > " + dir + "/" + file + "\n")
```

Comment:
I switched up the heredoc to avoid bash trying to interpolate things inside the VPN bootstrap script